cli/cmd/root.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// These are shared between all the commands
+var (
+	controller string
+	timeout    int
+)
+
+func init() {
```

Comment:
Probably more of a golang question, but is there an advantage to doing this in an init() rather than just in a normal func?